[CI] SnapshotLifecycleRestIT.testBasicTimeBasedRetention failing

<!--
Please fill out the following information, and ensure you have attempted
to reproduce locally
-->

**Build scan**: https://gradle-enterprise.elastic.co/s/sluoev3ttfqtm/tests/:x-pack:plugin:ilm:qa:multi-node:javaRestTest/org.elasticsearch.xpack.slm.SnapshotLifecycleRestIT/testBasicTimeBasedRetention?top-execution=1

**Repro line**: `./gradlew ':x-pack:plugin:ilm:qa:multi-node:javaRestTest' --tests "org.elasticsearch.xpack.slm.SnapshotLifecycleRestIT.testBasicTimeBasedRetention" -Dtests.seed=9C07192018733E66 -Dtests.locale=nl-BE -Dtests.timezone=America/Guatemala -Druntime.java=11`

**Reproduces locally?**: didn't try

**Applicable branches**: master

**Failure history**: https://gradle-enterprise.elastic.co/scans/tests?tests.container=org.elasticsearch.xpack.slm.SnapshotLifecycleRestIT&tests.test=testBasicTimeBasedRetention
<!--
Link to build stats and possible indication of when this started failing and how often it fails
<https://build-stats.elastic.co/app/kibana>
-->
**Failure excerpt**:

```
AssertionError: expected snapshot to exist but it does not: {"error":{"root_cause":[{"type":"snapshot_missing_exception","reason":"[time-based-retention-repo:snap-jtopkav2rvqjzxd0eeljtw] is missing"}],"type":"snapshot_missing_exception","reason":"[time-based-retention-repo:snap-jtopkav2rvqjzxd0eeljtw] is missing"},"status":404} |  
```

--- a/x-pack/plugin/ilm/qa/multi-node/src/javaRestTest/java/org/elasticsearch/xpack/slm/SnapshotLifecycleRestIT.java
+++ b/x-pack/plugin/ilm/qa/multi-node/src/javaRestTest/java/org/elasticsearch/xpack/slm/SnapshotLifecycleRestIT.java
@@ -437,7 +437,7 @@ public class SnapshotLifecycleRestIT extends ESRestTestCase {
             } catch (ResponseException e) {
                 fail("expected snapshot to exist but it does not: " + EntityUtils.toString(e.getResponse().getEntity()));
             }
-        });
+        }, 60, TimeUnit.SECONDS);
 
         // Run retention every second
         ClusterUpdateSettingsRequest req = new ClusterUpdateSettingsRequest();

